lib/inspec/base_cli.rb

```diff
@@ -8,6 +8,10 @@
 
 module Inspec
   class BaseCLI < Thor
+    class << self
+      attr_accessor :command
```

Comment:
This seems a tad dangerous (adding a class instance variable named simply 'command' seems like begging for a name collision when you are subclass of Thor), and I don't see it being used (I do see it being set, but not read).  Is it needed?

Reply:
Ahh, found it in the local fetcher.  At least consider renaming it to `inspec_cli_command` or something, to avoid any potential collision with Thor.

Reply:
Great catch. I'll update to use `inspec_cli_command`.